src/context/Cargo.toml

```diff
@@ -24,11 +24,11 @@ path = "../ecs/"
 version = "0.1.0"
 
 [dependencies]
-gfx = "0.11"
-gfx_core = "0.3"
-gfx_device_gl = "0.10"
-gfx_window_glutin = "0.11"
-glutin = "0.5"
+gfx = { git = "https://github.com/gfx-rs/gfx" }
+gfx_core = { git = "https://github.com/gfx-rs/gfx" }
+gfx_device_gl = { git = "https://github.com/gfx-rs/gfx" }
+gfx_window_glutin = { git = "https://github.com/gfx-rs/gfx" }
```

Comment:
Thanks, I wanted to use the newly published 0.12.2 but I noticed that it depends on the outdated 0.4 version of `gfx_core`.